sky/shell/platform/mojo/dart_tracing.cc

```diff
@@ -0,0 +1,172 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
```

Comment:
The purpose is the same. We can put this under mojo/common - but since this depends on dart/, we would need to roll dart into engine in lockstep with mojo. Or maybe dart_tools_api.h is stable enough that this will not be a problem in practice?